docs/reference/announcements.md

```diff
@@ -10,6 +10,10 @@ Release date: 9th of January 2024
 
 End of maintenance: 9th of July 2025
 
+:::caution
+For existing clusters we recommend updating to `8.4.3` directly and not `8.4.0` due to issues in [Form Linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#using-the-link-button) to user tasks which can cause wrong forms linked to user tasks.
+:::
+
```

Comment:
I would start by giving the Form Link feature context first, as for other users it doesn't matter.

```suggestion
:::caution
Customers making use of the [Form Linking](/components/modeler/web-modeler/advanced-modeling/form-linking.md#using-the-link-button) feature must update to `8.4.3` immediately as the `8.4.0`, `8.4.1` & `8.4.2` releases suffer from a [issue 16311](https://github.com/camunda/zeebe/issues/16311) that causes random user forms to get linked with any user tasks. Effectively corrupting the user ask instance and process instance affected by that bug.
:::

```

wdyt?

Reply:
That is definitely a better one, thanks @megglos!